canonical-websites/cloud-init.io
# Add global nav to cloud-init.io

## Done

- added the new Canonical global nav
- changed the banner colour
- upgraded Vanilla to 1.8.0

## QA

- download this branch
- run the site ./run
- see that the new nav is present and looks good - http://0.0.0.0:8009/


## Issue / Card

Fixes #85

## Screenshots

![image](https://user-images.githubusercontent.com/441217/46870434-7e6dbc00-ce26-11e8-8d65-5067072cd390.png)
